### ISSUE TITLE
Images will not load from text file

### ISSUE THREAD

**Issue by Elric888** [nhmkdev/cardmaker#119]
When following the tutorial provided here for creating a basic deck, and using the linked .zip, I was able to successfully populate text elements from the .csv, but not any of the images. All text elements functioned, but no images functioned. (Changing the image elements from image to text caused the appropriate image file names from the .csv to appear.) I placed a local file path for one of the images into the appropriate image element definition (as suggested here to another user), and this caused the specified image to load. The trouble with this is that if I cycle through the various cards, I will always end up with only the image whose file path I provided, thereby bricking the entire element. 

Using the Issues feature returned "Image file not found" messages for all every image that the test cards would use.

I am running version 1.3.0.0 [UNSTABLE] V.A7 on Windows.

I hope this report finds you well and helps to resolve the issue in future. (I was really hoping to avoid using PS variables to make cards again!)

**Comment by nhmkdev** [comment]
Thanks for reporting the issue. The full-path-working while relative-path-not-working issue sounds familiar.

A couple of additional questions:

- Which version of windows are you running?
- Where do you have the project and images folders? I have seen some prior issues related to files within the users (and desktop) folder.

I am able to load the included sample project (with images) with the folder: `C:\Users\[myuser]\Desktop\CardMaker_v.1.3.0.0-unstable.v.a7` but that could just be my luck!

Where are the images in relation to your `.cmp` file? 
- For the sample project the images are in a folder next to the `sample_project.cmp` called `images` 
- The reference file then has entries like this for the images: `images/biohazard.png`

**Comment by Elric888** [comment]
Thank you for your swift response! I'll do my best to provide some information pertinent to your follow-up questions:

I'm running Windows 10 (should be up to date with patching).

The folder structure looks like this: This PC\New Volume (D:)\Card Maker\Test Deck

The .cmp saved to the Card Maker folder by default--I did not try to fiddle with this position a tall. Meanwhile the Images and .csv are in the Test Deck folder.

Also the .zip I was referring to has the attack, sneaker, weakling, etc images, not the biohazard, although know that I am looking more deeply, I also see the folder structure and files you mention. Should I always assume to build a structure with a folder containing the .csv and .cmp and then a second folder therein that contains only the images files, as per the sample you reference here? Perhaps I have naively allowed things to save wherever they would and needed instead to direct the process more closely.

**Comment by Elric888** [comment]
Update:

After I noticed where my .cmp had ended up, I strongly suspected some user error on my part. I rebuilt the test project with the .csv and .cmp in a folder with all of the images in a second folder inside of the first. This approach got the same undesirable result as previously reported. However, when I placed all of the project assets (.csv, .cmp, and images) loose in a _single_ folder, everything loaded in perfectly. This is the same sort of convention that one usually wants to use with PS variables as well, hence why I thought to try it out.

Thanks for your response, had you not asked those questions I would not have thought to go look at he file structure more closely. May I suggest including a caution in the tutorial materials concerning the potential need to build a project in one folder with all assets floating loose therein? The example structures have some intervening folders, and if one naively copies that, as I did, they might stumble across this issue.

Thanks again!
